packages/field-base/src/delegate-focus-mixin.js

```diff
@@ -48,11 +48,19 @@ export const DelegateFocusMixin = dedupingMixin(
            * By default, the host element does not have tabindex attribute. Instead, `focusElement` should have it.
            * Toggling `tabindex` attribute on the host element propagates its value to `focusElement`.
            *
+           * @override
            * @protected
            */
           tabindex: {
-            type: Number,
-            value: undefined
+            type: Number
```

Comment:
This property can be probably removed now when we don't need to reset `value: undefined`

Reply:
Done.